faq/client.md

```diff
@@ -0,0 +1,237 @@
+<!--
+------------------------------------------------------------------------
+  layout: default
+  hide: true
+  tags: \[faq\]
+  title: Sawtooth FAQ - Client
+  permalink: /faq/client/
+  \# Copyright (c) 2018, Intel Corporation.
+    \# © Copyright 2020, Dr Kent G LAU, <kenty@kenty.com>.
+  \# Licensed under Creative Commons Attribution 4.0 International License
+  \# <https://creativecommons.org/licenses/by/4.0/>
+  \-\
+  -->
+
+# Sawtooth FAQ: Client
+  
+---
```

Comment:
These division lines should be omitted in the markdown. (same for other instances)